sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharpExplodedGraph.cs

```diff
@@ -440,6 +440,16 @@ protected override void VisitInstruction(ExplodedGraphNode node)
                     }
                     break;
 
+                case SyntaxKind.BaseConstructorInitializer:
+                case SyntaxKind.ThisConstructorInitializer:
+                    var constructorInitializer = (ConstructorInitializerSyntax)instruction;
+                    if (constructorInitializer.ArgumentList != null)
+                    {
+                        var ctorInifializerArgumentsCount = constructorInitializer.ArgumentList.Arguments.Count;
+                        newProgramState = newProgramState.PopValues(ctorInifializerArgumentsCount);

```

Comment:
Since the initializer arguments were added to the CFG, we generated symbolic values, which are on the stack. To ensure integrity of the stack we need to pop the values when we are walking the initializer instruction, as if we did a method invocation.